Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Eel/ElasticSearchQueryBuilder.php

```diff
@@ -449,6 +468,28 @@ public function fulltext($searchWord) {
 	}
 
 	/**
+	 * Wrap search term matches found in the result for highlighting purposes. Only available for fulltext searches.
+	 *
+	 * @param integer $fragmentSize
+	 * @param integer $fragmentCount
+	 * @return QueryBuilderInterface
+	 */
+	public function highlight($fragmentSize = 150, $fragmentCount = 2) {
+		$this->request['_source'] = array('__fulltext');
```

Comment:
for us no highlight was done unless the _source parameter was set, it made no sense though.. let's try without it and I'll test it in our environment and see if it works this fine :)